src/Database/Adapter/Mongo.php

```
@@ -0,0 +1,2964 @@
+    /**
+     * Update Document
+     *
+     * @param Document $collection
+     * @param string $id
+     * @param Document $document
+     * @param bool $skipPermissions
+     * @return Document
+     * @throws DuplicateException
+     * @throws DatabaseException
+     */
+    public function updateDocument(Document $collection, string $id, Document $document, bool $skipPermissions): Document
```

PHPMD (2.15.0)

Avoid unused parameters such as '$skipPermissions'. (undefined)

(UnusedFormalParameter)